sky/backends/backend_utils.py

```diff
@@ -138,36 +137,37 @@ def fill_template(template_name: str,
         output_path = str(output_path)
     output_path = os.path.abspath(output_path)
 
-    # Runtime files handling
-    #
-    # List of runtime files to be uploaded to cluster:
-    #   - yaml config (for autostopping)
-    #   - wheel
-    #   - credentials
-    # Format is {dst: src}.
```

Comment:
Can we keep this in L152 or L161?  So that we know what file mounts are being bundled. Can also mention "runtime" files in that docstr.

Reply:
Ahh, added back. Thanks!